pending/notepad++.xml

```diff
@@ -1,27 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
+    @author    Christian Foellmann <foellmann@foe-services.de>
+    @copyright Copyright (c) 2015, Christian Foellmann
+    @license   http://www.gnu.org/licenses/gpl-3.0.html GPLv3
+    @package   BleachBit\CleanML
 
-    BleachBit
-    Copyright (C) 2015 Andrew Ziem
-    http://bleachbit.sourceforge.net
-
-    This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License
-    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-	@app     Notepad++
-	@link    http://notepad-plus-plus.org/
-	@os      Windows
-	@version 0.1.0
+    @app     Notepad++
+    @link    http://notepad-plus-plus.org/
+    @os      Windows
+    @version 0.1.0
```

Comment:
I guess this is a version number for a cleaner? It feels redundant since we already have git history/commit hashes.